traveler-swift-core/TravelerKit/Models/BookingItemSearchFilters.swift

```diff
@@ -12,10 +12,20 @@ import Foundation
  A model representing different ways in which the items in `BookingItemSearchResult` can be filtered
  */
 public struct BookingItemSearchFilters {
+    /// Keywords
+    public var text: String?
     /// Item category
     public var categories: [ProductItemCategory]?
```

Comment:
Add TODO comment here that ProductItemCategory should be refactored. I'll create a jira ticket for this task.